CIP-0072/version_1_offchain.json

```diff
@@ -11,63 +16,92 @@
       },
       "projectName": {
         "type":"string",
-        "description": "A project name, e.g. My dApp."
+        "description": "A project name, e.g. My dApp.",
+        "maxLength": 40
       },
       "link": {
         "type":"string",
         "description": "Website presenting a dApp.",
-        "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})"
-      },      
+        "pattern": "(https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|www\\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\\.[^\\s]{2,}|https?:\/\/(?:www\\.|(?!www))[a-zA-Z0-9]+\\.[^\\s]{2,}|www\\.[a-zA-Z0-9]+\\.[^\\s]{2,})",
+        "maxLength": 200
+      },
       "logo": {
-        "type":"string",
-        "description": "URL to the logo or the base64 encoded image."
+        "type": "string",
+        "description": "URL to the logo. Minimum resolution: 512x512px, supported formats: PNG/JPG/SVG, maximum file size: 0.5 MB",
+        "maxLength": 1000
       },
       "categories": {
         "type":"array",
         "items": {
           "type": "string",
-          "enum":["Games", "DeFi", "Gambling", "Exchanges", "Collectibles", "Marketplaces", "Social", "Other"]
+          "enum": ["DeFi", "Development", "Education", "Games", "Identity", "Marketplace", "NFT", "Other", "Security"]
```

Comment:
@danielmain that's almost exactly what I thought, but it seems to be addressed by https://github.com/cardano-foundation/CIPs/pull/612#discussion_r1369900011.

Reply:
I will skip that for now :)

Reply:
Hey everyone, 

Thank you a lot for your comments! agree that we have room to improve this suggested DApp category list, and all your feedback on it is much appreciated.

As the comment linked above outlines, this currently suggested list of DApp categories is just an initial one to get started and have something stable for initial implementation proposes for now. Once there is an implementation of this CIP, there will be more feedback coming to elaborate this DApp category list.

Reply:
@rphair do you think we can approve this PR and merge it with that list of categories 🙏🏻 and later on add more ?

Reply:
@danielmain I'm in favour of that & have already passed it about 1 week ago (https://github.com/cardano-foundation/CIPs/pull/612#pullrequestreview-1710779814).

We still need 1 more approval from @Ryun1 @Crypto2099 @KtorZ @SebastienGllmt before it can be merged (I invited them for review as soon as it was clear all questions were answered except https://github.com/cardano-foundation/CIPs/pull/612#discussion_r1377168198).

Because of limited availability during the Summit I also added it to the next CIP meeting agenda 6 days from now, to make sure it doesn't get missed, and will try to catch up other editors on these issues if necessary so it can be merged at meeting time if not sooner.